src/design-system/HelpButtonWithTooltip.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from "styled-components";
+
+import Colors from "./Colors";
+import Tooltip from "./Tooltip";
+
+const HelpButton = styled.span`
+  background: ${Colors.forest};
+  color: white;
+  border-radius: 14px;
+  line-height: 12px;
+  width: 16px;
+  display: inline-block;
+  text-align: center;
+  font-size: 9px;
+
+  padding: 2px;
+`;
+
+const HelpButtonWithTooltip: React.FC = (props) => {
+  return (
+    <Tooltip content={props.children}>
+      <HelpButton>?</HelpButton>;
```

Comment:
This is putting an accidental semi-colon after every tooltip.

Reply:
wow, it was so tiny that I think I just kept on thinking there was dust on my screen. whoops